### PR TITLE
Issue #73-Update README.md in frontend

### PR DESCRIPTION
**Pull request by 5hr1ganesh**
Added a more Descriptive breakdown of the frontend section of the repository, to the existing readme file of the frontend repository.

### CODE REVIEW

**Comment by 5hr1ganesh** [comment]
@euanwm @mountclear can you please review this pr?

File: frontend/README.md [hunk 1]
```diff
@@ -1,8 +1,39 @@
...
+
+The frontend code is organized into the following directories:
+* api: Contains the TypeScript code for fetching data from the OpenWeightlifting backend API
+* components: Contains the reusable React components that make up the user interface of the application.
+* config: Contains configuration files for the application, such as the fonts and site colors.
+* layouts: Contains the layout components for the application, such as the header and footer.
+* pages: Contains the React components for the individual pages of the application, such as the leaderboard page and the lifter page.
+* styles: Contains the CSS files for the application.
+
+### Entry Point
```
Comment (euanwm) [inline frontend/README.md:17]: This section can be removed.

File: frontend/README.md [hunk 2]
```diff
@@ -1,8 +1,39 @@
...
+
+The entry point for the React application is the `App.tsx` file. This file contains the root component for the application.
+
+### Serving the Application
```
Comment (euanwm) [inline frontend/README.md:21]: This section can also be removed.

Comment (5hr1ganesh) [reply frontend/README.md]: ok 👍

**Review by euanwm — APPROVED** [review]
Approved.

**Comment by euanwm** [comment]
Closes #253

**Pull request merged**